pipeline/lib/flood_model/settings.py

```diff
@@ -61,15 +61,16 @@
     print('No secrets file found.')
 
 DATALAKE_API_VERSION = "2018-11-09"
-#IBF_URL='https://ibf-test.510.global/api/'
+# IBF_URL='https://ibf-test.510.global/api/'
+# IBF_PASSWORD = 'password'
 
 ######################
 ## COUNTRY SETTINGS ##
 ######################
 
 # Countries to include
 #COUNTRY_CODES = ['ETH','ZMB','KEN','UGA'] #
-COUNTRY_CODES = ['SSD'] # 
+COUNTRY_CODES = ['MWI'] # 
```

Comment:
please change COUNTRY_CODES  list to include all the countries 
COUNTRY_CODES = ['ETH','ZMB','KEN','UGA','MWI']